base/common/src/main/java/com/netscape/certsrv/util/JSONSerializer.java

```diff
@@ -1,19 +1,51 @@
 package com.netscape.certsrv.util;
 
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.module.jaxb.JaxbAnnotationIntrospector;
 
 /**
  * Implement this interface to provide default methods to serialize an object to/from JSON
  */
 public interface JSONSerializer {
 
+    default String toJSON(boolean requiresPrivateFieldAccess) throws Exception {
+        if (requiresPrivateFieldAccess) {
+            ObjectMapper mapper = new ObjectMapper();
+            return mapper.enable(SerializationFeature.INDENT_OUTPUT)
+                    .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
```

Comment:
Should we just enable field visibility in all cases? I don't see any downside with that.

Reply:
I was trying to preserve the existing behaviour as much as possible but if you think it is harmless to do this then I will spare us the additional methods.

Reply:
Hmm.. I think this is only needed to access the private `RequestStatus.label`:
https://github.com/dogtagpki/pki/blob/master/base/common/src/main/java/com/netscape/certsrv/request/RequestStatus.java#L114

Should we just make `label` public? Or maybe make the constructors public? Not sure if it will work though.

Reply:
We could make it public, I'll try. Better to break encapsulation in one place than everywhere by default I guess.